README-ANSIBLE.md

```diff
@@ -147,4 +147,13 @@ ansible-playbook ansible/main.yml -t istio -e '{"cluster_url": "host:ip"}'
 - User already has the oc binary pointing to a running Minishift cluster and wants to install Istio with settings other than the default
 ```bash
 ansible-playbook ansible/main.yml -t istio -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "jaeger": true, "delete_resources": true}}'
-```
\ No newline at end of file
+```
+
+- User already has the oc binary pointing to a running Minishift cluster and wants to customize the Istio addons
```

Comment:
This is not mandatory to have minishift. Prereq is to be authenticated with an openshift instance where namespace of istio corresponds to what is defined within ansible playbook. Can you adapt the sentence please ?

Also this text is ambiguous too "User already has the oc binary pointing to a running Minishift cluster" oc client doesn't point to minishift ...

Reply:
Sure, I will update it